examples/app-python/example_app.py

```diff
@@ -156,6 +215,7 @@ def new():
     app.mount_sequence("UARTLoopback", ctl.handle_uart_loopback)
     app.mount_sequence("StageFile",ctl.handle_stage_filedownload)
     app.mount_sequence("PowerControl", ctl.handle_power_control)
+    app.mount_sequence("TestCanBus", ctl.handle_test_can_bus)
```

Comment:
Minor suggestion - capitalize CAN here so it is clear

Reply:
Updaed sequence name